### ISSUE TITLE
AppJoint repository size

### ISSUE THREAD

**Issue by enginegl** [PrototypeZ/AppJoint#1]
I just cloned AppJoint repository and found out that for some reason it contains gradle-4.1-all.zip (almost 86 MB).
Is it there by mistake?

**Comment by PrototypeZ** [comment]
Oh, sorry for that, I commit it by mistake, I will remove it from repository, thank you :)

**Comment by enginegl** [comment]
The bad thing here is that GIT remembers everything. So even after the fix everybody will get:

▶ git clone https://github.com/PrototypeZ/AppJoint.git
Cloning into 'AppJoint'...
remote: Enumerating objects: 1076, done.
remote: Counting objects: 100% (1076/1076), done.
remote: Compressing objects: 100% (435/435), done.
remote: Total 1490 (delta 319), reused 969 (delta 224), pack-reused 414
Receiving objects: 100% (1490/1490), 83.64 MiB | 3.04 MiB/s, done.
Resolving deltas: 100% (418/418), done.

**Comment by PrototypeZ** [comment]
You are right, I'm working on it to remove this large file from git history. Thanks : )

**Comment by razerdp** [comment]
maybe this command work perfect

git filter-branch --tree-filter 'rm -rf gradle/wrappe/gradle-4.1-all.zip' HEAD
git reflog expire --expire=now --all
git gc --prune=now
git push origin --force --all

rebuild all commit tree for delete gradle-4.1-all.zip
